applications/OptimizationApplication/python_scripts/utilities/opt_projection.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, parameters: Kratos.Parameters, _):
         parameters.ValidateAndAssignDefaults(default_settings)
         pass
 
-    def SetProjectionSpaces(self, _: list[float], __: list[float]) -> None:
+    def SetProjectionSpaces(self, _: 'list[float]', __: 'list[float]') -> None:
```

Comment:
or update to python 3.10 :eyes:

Reply:
I would prefer to merge this, having windows back online, and then try to move to 3.10 tbh

Reply:
yes I know, was half teasing / half FYI